template-helpers/hbs-helpers.js

```diff
@@ -21,6 +21,25 @@ function getSupportedLocales(args) {
 }
 
 
+function englishInAcceptLanguages(args) {
+  const acceptedLanguages = args.data.root.req.acceptsLanguages();
+  return acceptedLanguages.some(locale => locale.startsWith("en"));
+}
```

Comment:
Confirmed by hacking my `intl.accept_languages` in about:config

![Erweiterte_Einstellungen](https://user-images.githubusercontent.com/557895/95626748-b87ef200-0a2f-11eb-8517-e74e7cb64029.png)